pkgs/development/python-modules/pytest-twisted/default.nix

```diff
@@ -18,7 +18,7 @@ buildPythonPackage rec {
     sha256 = "sha256-IJv1pkUs+/th3o8BWQLBTsgSZACRFQcHS7LuTOjf4xM=";
   };
 
-  buildInputs = [
+  propagatedBuildInputs = [
```

Comment:
We on purpose do not propagate pytest (as sort of an exception) so we can handle multiple pytest + plugins which we at times need to.

Reply:
shoot, thats a holdover from before i added `pythonImportsExtrasCheck`